pkg/steps/cluster_claim.go

```diff
@@ -175,7 +175,11 @@ func (s *clusterClaimStep) acquireCluster(ctx context.Context, waitForClaim func
 	return claim, nil
 }
 
-func getHiveSecret(src *corev1.Secret, name, namespace string) (*corev1.Secret, error) {
+func namePerTest(name, testName string) string {
+	return fmt.Sprintf("%s-%s", testName, name)
+}
```

Comment:
I'm worried about the length - secret names have limited length. Job names are sometimes fairly long and our suffix is too. I think we can just prune the testname to be sure we can append a suffix with known length safely?

Reply:
Updated.
Also replace "." by "-".